netlify.toml

```diff
@@ -0,0 +1,17 @@
+# This configuration is for the standalone version of blockbuilder-search.
+# Used with netlify-cli tool
+# https://www.netlify.com/docs/netlify-toml-reference/
+[build]
+  command = 'npm run parcel-build'
```

Comment:
could we update this to use yarn? https://www.netlify.com/blog/2016/11/01/yarn-support-on-netlify/

`yarn parcel-build`